sfge/src/test/java/com/salesforce/rules/unusedmethod/InstanceMethodsTest.java

```diff
@@ -142,7 +142,6 @@ public void instanceInvokedBySubclass_expectNoViolation(
      * used.
      */
     @Test
-    @Disabled // TODO: FIX AND ENABLE THIS TEST
```

Comment:
This test was incidentally fixed by the changes related to `super`. I think it corresponds to a different bug, and if so then we can mayube close that one as Could Not Reproduce.